.github/workflows/main.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           python -VV
           python -m site
           python -m pip install --upgrade pip setuptools wheel
-          python -m pip install --upgrade coverage[toml] tox tox-gh-actions
```

Comment:
So removing `tox-gh-actions` also works but it's not as explicit as using it.

The tests basically try to run all `tox` defined environments but ends up running only the desired one since each worker in the matrix installs a specific Python version (not 100% true since it looks like 3.10 is in all the base images).

Reply:
@c0state I agree with the premise here in that removing another dependency and keeping the `tox` setup as boilerplate as possible is ideal.  In this setup, each matrix does run the corresponding tests for the pertinent python version, but it also runs the 310base as well each time.  I suspect the reason why is that the github runners come pre-installed with python 3.10, which is why the 3.10 env erroneously gets run for all the matrices:

https://github.com/actions/runner-images/blob/ubuntu22/20240901.1/images/ubuntu/Ubuntu2204-Readme.md#language-and-runtime

I think we should either figure out why to parameterize the matrix runs to only run the pertinent env if we go with this approach.  Or we could use the tox-gh-actions, which I believe does exactly that under the hood.

Reply:
Makes sense. Will close and keep using the plug-in for now